src/main/java/net/dv8tion/jda/api/interactions/commands/build/OptionData.java

```diff
@@ -47,6 +47,11 @@
      * The maximum length the name of an option can be.
      */
     public static final int MAX_NAME_LENGTH = 32;
+    
+    /**
+     * The maximum length of the name of some OptionData names
+     */
+    public static final int MAX_NAME_OPTION_LENGTH = 100;
```

Comment:
This seems to only apply to choices, so maybe it should be `MAX_CHOICE_NAME_LENGTH`?